src/style.css

```diff
@@ -1,3 +1,77 @@
+/* stylelint-disable font-family-name-quotes */
+
+:root {
+  --main-blue: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  background-color: gray;
+}
+
+header {
```

Comment:
Don't fix container size (if there is no such a requirement). Here the `max-width: 100%;` is unnecessary for the `header` since block-level elements like `header` naturally occupy 100% of the width of their parent element, unless specified otherwise.